frontend/src/app/modules/bookings/bookings-page/bookings-page.component.sass

```diff
@@ -25,6 +25,8 @@ p, span
     color: $pale-sky
 
 .item
+    min-width: 300px
+    width: 300px
```

Comment:
why in px?

Reply:
I think this component needs to be refactored, but it is a little hot fix before the demo, and will be rewritten in the redesign

Reply:
Fixed

Reply:
it would be better to use %